gravitee-apim-console-webui/src/management/application/details/user-group-access/transfer-ownership/application-general-transfer-ownership.component.spec.ts

```diff
@@ -194,21 +196,8 @@ describe('ApplicationGeneralTransferOwnershipComponent', () => {
 
     const groupSelect = await loader.getHarness(MatSelectHarness.with({ selector: '[formControlName="groupId"]' }));
     await groupSelect.open();
-
     const options = await groupSelect.getOptions();
-    expect(options.length).toBe(2);
-
-    const transferButton = await loader.getHarness(MatButtonHarness.with({ text: 'Transfer' }));
-    await transferButton.click();
-
-    const dialog = await rootLoader.getHarness(MatDialogHarness);
-    await (await dialog.getHarness(MatButtonHarness.with({ text: 'Transfer' }))).click();
-
-    const req = httpTestingController.expectOne({
```

Comment:
why you remove this part of test ? 
Without it, you won't be able to test that the transfer has been requested.

Reply:
Done